src/common/signalk/SKUnits.h

```diff
@@ -32,3 +32,6 @@
 
 #define SKDegToRad(x) x * 2 * M_PI / 360
 #define SKRadToDeg(x) x * 360.0 / (2 * M_PI)
+
+#define SKMsToKnots(x) x * 3600.0 / 1852.0
```

Comment:
Just to be consistent with the other conversions I think we should say `SKMsToKnot()` here (instead of knot**s**).

Reply:
yes of course, you are right!